src/Query/BookingQueryInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace RebelCode\Diary\Query;
+
+/**
+ * Any object that can describe a bookings query.
+ *
+ * @since [*next-version*]
+ */
+interface BookingQueryInterface
+{
+    /**
+     * Gets the query condition.
+     *
+     * @return \Dhii\Espresso\BooleanExpressionInterface
+     */
+    public function getCondition();
```

Comment:
Ah! I didn't think about such cases. You're right - another interface _is_ needed. Thanks